### ISSUE TITLE
How to change the learning rate? 

### ISSUE THREAD

**Issue by Oldyin1012** [open-mmlab/mmdetection#4276]
retinanet,  I tryed to change lr and warmup ratio in schedule_1x.py,  but when training the lr is not change and the loss became nan

**Comment by ZwwWayne** [comment]
Please use the `Error template` or `reimplementation issues` to describe your modification and the issue you met.
